microk8s-resources/actions/disable.cilium.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Disabling Cilium"
+
+if [ -f "${SNAP_DATA}/bin/cilium" ]
+then
+  "$SNAP/kubectl" "--kubeconfig=$SNAP_DATA/credentials/client.config" delete -f "$SNAP_DATA/actions/cilium.yaml"
```

Comment:
Disabling left this behind `kube-system   pod/cilium-52mcz   0/1     Terminating`. This is probably because we do not wait for the pod to finish termination and we restart kubelet immediately. Could we do an active wait as we do here: https://github.com/ubuntu/microk8s/blob/feature/insecure-port/microk8s-resources/actions/disable.dns.sh#L26 The reason I am asking this is because our tests will hang waiting for the pod to terminate :(